documentation/abnormal_termination/README.md

```diff
@@ -39,11 +43,53 @@ To find out if the process exited because of a `process.exit()` call, follow [Us
 
 ### Process exits with Native Stack Trace
 
-TBD
+Some Node.js application uses native modules to re-use existing components or
+performance optimize specific areas. A common example for native modules are
+database drivers. In the case our native module has an issue it can crash the
+Node.js application which will generate a native stack trace.
 
-### Process exits with unmeaningful Stack Trace
+You can find the most common scenarios for native crashes here:
 
-TBD
+1. Native module crashes due bug
+2. Native module crashes due memory leak (native memory leak)
+3. Bugs in Node.js runtime and dependencies, for example:
+   - V8 checks, when input data checking to v8 APIs is missing can lead to a crash
+   - Access garbage collected data
+   - Call v8 APIs without HandleScope
+   - Call v8 APIs from wrong Thread
+   - Use native API compiled on linux using glibc on Alpine which uses Musl
```

Comment:
- accessing NULL pointer
 - attempting to process invalid file descriptors
?

Reply:
Thanks, I added them